src/utils/url.ts

```diff
@@ -61,6 +68,18 @@ export class Url extends URL {
       .some((queryParameter) => contentApiParameters.includes(queryParameter))
   }
 
+  public isProbablyUuid(): boolean {
+    // TODO: Only after we have deleted the old alias system we know for sure
+    // which paths resolve to an uuid. So far we can do the following:
+    return this.isFrontendSupportedAndProbablyUuid()
+  }
+
+  public isFrontendSupportedAndProbablyUuid(): boolean {
+    return !pathsFrontendSupportsWithoutUuids.some(
+      (regex) => regex.exec(this.pathname) !== null
+    )
+  }
```

Comment:
Sorry if I am mistaken here, but couldn't you just dissolve the double negation?

Reply:
Thanks, have implemented it